libevm/VM.h

```diff
@@ -69,14 +69,14 @@ class VM: public VMFace
 
 private:
 
-	u256* m_io_gas = 0;
+	u256* io_gas = 0;
+	uint64_t m_io_gas = 0;
```

Comment:
I doubt it, but I switched the code to 64 bits months before we finally decided on 63 bits.